packages/bierzo-wallet/src/routes/payment/components/ConfirmPayment/index.tsx

```diff
@@ -1,22 +1,34 @@
-import { faUser } from "@fortawesome/free-regular-svg-icons";
+import { faCopy } from "@fortawesome/free-regular-svg-icons";
```

Comment:
There is a little different icon used for coping. https://github.com/iov-one/ponferrada/blob/fac359e7e8c34ba9a0d02a60cd8e665b519795c8/packages/bierzo-wallet/src/components/AddressesTable/index.tsx#L9